### PR TITLE
Ivysaur 90

### PR DESCRIPTION
**Pull request by Conner-H**

### CODE REVIEW

File: client/src/styles/styleDefaults.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  --appColourLightest: #b5c9e8;
+  --redColourLightest: #f7dfe4;
+  --successColour: #2ecc71;
+  --brighterSuccess: #11e268;
+  --failColour: #ff0000;
+  --appColourDarker: #0c2348;
+  --appColourDarkest: #051022;
+
+    /*  Fonts   */
+  --fontFamily: Helvetica, Arial;
```
Comment (camfiander) [inline client/src/styles/styleDefaults.css:16]: Is there a reason we're using Helvetica and Arial here without including the `openSans` font we defined below?

Comment (Conner-H) [reply client/src/styles/styleDefaults.css:16]: not really, I just didn't remove the variables when I brought over the styles page.

**Review by mmacdo52 — CHANGES_REQUESTED** [review]

File: client/src/page/loginPage.jsx [hunk 2]
```diff
@@ -41,6 +41,9 @@ function Login() {
       <Button variant="contained" onClick={testFunction}>
         Submit
       </Button>
+      <Button variant="contained" color="secondary" onClick={testFunction}>
```
[the remark this answers is not in the document]
Comment (Conner-H) [reply client/src/page/loginPage.jsx]: done, thought I got rid of it earlier, sorry

**Review by mmacdo52 — COMMENTED** [review]
I am not a big fan of the forms taking up the whole page width. Maybe we should discuss this before merging

**Review by mmacdo52 — APPROVED** [review]

**Pull request merged**